app/facades/github_search.rb

```diff
@@ -0,0 +1,26 @@
+class GithubSearch
+  def initialize(token)
+    @token = token
+  end
+
+  def display_repos
+    get_json('repos').take(5)
+  end
+
+  def display_following
+    get_json('following').take(5)
+  end
+
+  def find_github_resource(data_type, data)
```

Comment:
This takes the variable passed into the display methods i.e. display_repos and converts it into something we can instantiate a new object with such as Repo.new